### DISCUSSION TITLE
Bahas Dessign Patterns

### DISCUSSION THREAD

**Discussion by azman-aziz** [ngobrolin/web#21] [Ideas]
Beberapa Pertanyaan penunjang: 
- secara ideal design pattern ini di tentukan berdasarkan apayah? apakah dari base project yang mau di bangun, semisal pertimbanganya mau bikin system payment gateway nah apa cocok kalo kita pake design pattern berbasis service?
- kebanyakan tech company besar itu pake dessign pattern seperti apa?
- tips buat junior progammer yang mau mahamin design pattern ini seperti apa di luar komunikasi ke senior dan baca dokumentasi projectnya?

terimakasih :)

**Comment by rizafahmi** [comment]
Contohnya seperti ini ya @azman-aziz? https://refactoring.guru/design-patterns

**Comment by rizafahmi** [comment]
Akan kita bahas di sesi malam ini https://www.youtube.com/live/utmuHV8-MAU?feature=shared
